### PR TITLE
My Portfolio Mobile Version Skeleton Microverse Project Complete

### PR DESCRIPTION
**Pull request by David-Kasilia**
- After Merging the setup-project branch with the main branch I completed the following:
- Created the initialization-project branch to build the project code.
- Opened the folder Using VSCode.
- Completed the HTML code first
- Completed the CSS part.
- I tested the website by running the code to a live server and opened it with Mozilla Firefox.
- To ensure the Webpage was responsive and was in a mobile version skeleton I used the developer tools to see if it was responsive.
- Fixed the linters error on Html by the webhint linter
- Fixed the stylelint errors on CSS.
- Fixed Margin errors on HTML file.
- Used section container in the header.
- Changed the font to the text to roboto.
changes

-

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    <link rel="stylesheet" href="./stylesheet/style.css">
+</head>
+
+<body>
+
+    <nav>
+        <h4>David Kasilia Logo</h4>
+        <img class="menu" src="./images/menu_icon-removebg-preview.png" alt="Menu Icon">
+    </nav>
+
```
Comment (MeqdamAlqudah) [inline index.html]: - Please The section should have the margins (top, bottom, left, right) defined in the design

| Your Design | Figma design |
| ---------------- | ------------------ |
| image | image |

File: index.html [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    <article>
+        <h2>Software Developer</h2>
+    </article>
+  
+
+<div class="text">
+    <p>I am a software developer motivated with passion to work and provide the best product that will impress my clients.
+    My services include building mobile applications and websites. Explore My projects and if you like them feel free to contact me.</p>
+    </div>
+    <img class="line" src="./images/line-1.png" alt="Horizontal line">
```
Comment (MeqdamAlqudah) [inline index.html]: - [x] Please fix the margin and the padding of the website 
image


- [x] Please use section tag as a container of the headline section 
image

- [ ] Please I see that you didn't use font here use the same font-family as in figma design

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @David-Kasilia , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — APPROVED** [review]
### Hi @David-Kasilia, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- PR has a good description summary :heavy_check_mark:
 
> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**